recipes/cfchecker/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.0.5" %}
+
+package:
+    name: cfchecker
+    version: {{ version }}
+
+source:
+    fn: cfchecker-{{ version }}ceda.p1.tar.gz
+    url: https://pypi.python.org/packages/source/c/cfchecker/cfchecker-{{ version }}ceda.p1.tar.gz
+    sha1: 8709708c8c9417e91403a0e0aa539d3bae55f346
+    patches:
+        # Ensure our udunits2 lib is used on all platforms.
+        - ctypes.patch
```

Comment:
@jakirkham this packages are no longer supported and building them here will probably be a one time thing. The only reason I am doing this is because `xarray` can use `cdat_lite` to read `cdms2` objects and that is still used by many scientists in my group.